core/src/processing/core/PSurface.java

```diff
@@ -41,7 +41,7 @@
   // Background default needs to be different from the default value in
   // PGraphics.backgroundColor, otherwise size(100, 100) bg spills over.
   // https://github.com/processing/processing/issues/2297
-  static final Color WINDOW_BGCOLOR = new Color(0xDD, 0xDD, 0xDD);
+  static final Color WINDOW_BGCOLOR = new Color(0xAA, 0xAA, 0xAA);
```

Comment:
You're (obviously) welcome to change this sort of thing in your own copy of the code, but take care not to submit this in an unrelated pull request.